### PR TITLE
Clean up layout for the embed project page. 

### PR DESCRIPTION
**Pull request by MaxKorlaar**
## Description

Fixes broken layout for the project embed page. Currently the navigation and footer are hidden by listening to router events, this should be done otherwise, see #223.

## Type of change

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Check out this branch
2. Make a project embeddable. You must own or create a project for this
3. Go to the URL of that project's embed page and set your viewport width to a small amount, such as 600px or 400px.

## Link to issue

Closes: #223

### CODE REVIEW

File: src/app/modules/project/embed/embed.component.html [hunk 1]
```diff
@@ -71,10 +71,10 @@ <h1 class="project-title">{{ project.name }}</h1>
         <a class="project-tags__see-more-link" href="#">
           see more tags&hellip;
         </a>
-      </div>
+      </div>-->
```
Comment (wotwot563) [inline src/app/modules/project/embed/embed.component.html:74]: remove commented out html

**Comment by Brend-Smits** [comment]
Would it be possible to cut off long descriptions nicely with "desciasjidjwiurj...." using the ellipsis? This is how it currently looks for me: 
image

Perhaps we can also look into increasing the default height of the iframe to show the uploader, or hide it completely if it does not show up or if we don't want to increase the height.

File: src/app/modules/project/embed/embed.component.html (hunk 1 above)
Comment (MaxKorlaar) [reply src/app/modules/project/embed/embed.component.html:74]: begone

File: src/app/components/app-layout/app-layout.component.ts [hunk 2]
```diff
@@ -33,11 +33,12 @@ export class AppLayoutComponent implements OnInit {
   public isAuthenticated: boolean;
   public subscription: Subscription;
   public displayAlertContainer = false;
+  public contentOnlyLayout = false;
```
Comment (rubenb994) [inline src/app/components/app-layout/app-layout.component.ts:36]: Property could have a better name, without context I would not know what this property is used for. Suggested: displayContentWithoutLayout

File: src/app/components/app-layout/app-layout.component.ts [hunk 3]
```diff
@@ -33,11 +33,12 @@ export class AppLayoutComponent implements OnInit {
...
 
   public readonly dexGithubIssueUrl = 'https://github.com/DigitalExcellence/dex-frontend/issues/new/choose';
   public displayBetaBanner = true;
 
-  constructor(private authService: AuthService, private alertService: AlertService) { }
+  constructor(private authService: AuthService, private alertService: AlertService, private router: Router) { }
```
Comment (rubenb994) [inline src/app/components/app-layout/app-layout.component.ts:41]:
```suggestion
  constructor(
  private authService: AuthService, 
  private alertService: AlertService, 
  private router: Router) { }
```

**Comment by rubenb994** [comment]
As suggest within #223, the layout should be hidden with the use of a multi router setup

**Comment by MaxKorlaar** [comment]
> As suggest within #223, the layout should be hidden with the use of a multi router setup

Yeah, although I'm not sure if I'll be able to make it this week, the Delta site has some issues I also need to pick up.

**Comment by Brend-Smits** [comment]
> Yeah, although I'm not sure if I'll be able to make it this week, the Delta site has some issues I also need to pick up.

In that case, I think it's better to create a new detailed issue describing the enhancement and required changes. This issue can be picked up in the future. We need this change in production before Monday.

**Comment by rubenb994** [comment]
> > Yeah, although I'm not sure if I'll be able to make it this week, the Delta site has some issues I also need to pick up.
> 
> In that case, I think it's better to create a new detailed issue describing the enhancement and required changes. This issue can be picked up in the future. We need this change in production before Monday.

Agreed, can you open issue about this @MaxKorlaar?

**Pull request merged**